std/bigint.d

```diff
@@ -1761,3 +1761,111 @@ unittest
     auto b = immutable BigInt("123");
     assert(b == 123);
 }
+
+/// Extracts the quotient and remainder for the given dividend and divisor.
+void divMod(const BigInt dividend, const BigInt divisor, out BigInt quotient, out BigInt remainder) pure nothrow
```

Comment:
The common Phobos style is to avoid using overloads if not necessary as it overwhelms the user with documentation and it also allows to provide good error messages.
In your case as an additional benefit, it also allows you to get rid of hard-coding the attributes.
Use sth. like this instead when unifying the overloads:

```d
void divMod...
is(Dividend == BigInt) || is(Divisor == BigInt)
```